doc/examples/fruit_stand/generate/generate.go

```diff
@@ -48,17 +46,19 @@ func main() {
 	var commits int
 	var lines int
 	cmd := &cobra.Command{
-		Use:   os.Args[0],
 		Short: "Generate pfs traffic.",
 		Long:  "Generate pfs traffic.",
-		Run: pkgcobra.RunFixedArgs(0, func(args []string) error {
+		RunE: func(c *cobra.Command, args []string) error {
+			if len(args) != 0 {
+				return fmt.Errorf("invalid argument")
+			}
```

Comment:
I'd prefer to keep using `pkgcobra` for this, any particular reason you chose to drop it?